backend/course/crawler.py

```diff
@@ -0,0 +1,121 @@
+import requests
+from bs4 import BeautifulSoup as bs
+import xlrd
+
+
+class Crawler:
+    search_url = 'http://sugang.snu.ac.kr/sugang/cc/cc100.action'
+    excel_url = 'http://sugang.snu.ac.kr/sugang/cc/cc100excel.action'
+    semester_name = {
+        'U000200001U000300001': '1',
+        'U000200001U000300002': 'S',
+        'U000200002U000300001': '2',
+        'U000200002U000300002': 'W'
+    }
+    # srchOpenSchyy : year
+    # srchOpenShtm : semester
+    # srchOpenUpSbjtFldCd : area
+    # srchOpenSbjtFldCd : subarea
+    form_data = {
```

Comment:
This `form_data` isn't needed in crawling logic and unnecessarily long. I prefer making new `course/format.py` file and putting `form_data` there.